runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsConfig.java

```diff
@@ -88,43 +88,32 @@ default boolean showHighlightedOnly()
 	}
 
 	@ConfigItem(
-		keyName = "showGEPrice",
-		name = "Show Grand Exchange Prices",
-		description = "Configures whether or not to draw GE prices alongside ground items",
+		keyName = "dontHideUntradeables",
+		name = "Do not hide untradeables",
+		description = "Configures whether or not untradeable items ignore hiding under settings",
 		position = 4
 	)
-	default boolean showGEPrice()
+	default boolean dontHideUntradeables()
 	{
 		return true;
 	}
 
 	@ConfigItem(
-		keyName = "showHAValue",
-		name = "Show High Alchemy Values",
-		description = "Configures whether or not to draw High Alchemy values alongside ground items",
+		keyName = "priceDisplayMode",
+		name = "Price Display Mode",
+		description = "Configures what price types are shown alongside of ground item name",
 		position = 5
 	)
-	default boolean showHAValue()
+	default PriceDisplayMode priceDisplayMode()
 	{
-		return false;
-	}
-
-	@ConfigItem(
-		keyName = "showMenuItemQuantities",
-		name = "Show Menu Item Quantities",
-		description = "Configures whether or not to show the item quantities in the menu",
-		position = 6
-	)
-	default boolean showMenuItemQuantities()
```

Comment:
Why was this config option removed?

Reply:
You were asking about that already in original PR and I explained it there